frontend/package.json

```diff
@@ -13,32 +13,32 @@
   "dependencies": {
     "@bcgov/bc-sans": "^1.0.1",
     "axios": "^0.21.1",
-    "core-js": "^3.6.4",
-    "keycloak-js": "^9.0.2",
-    "vue": "^2.6.11",
+    "core-js": "^3.10.0",
+    "keycloak-js": "^9.0.3",
+    "vue": "^2.6.12",
     "vue-json-csv": "^1.2.12",
-    "vue-router": "^3.1.6",
-    "vuetify": "^2.2.11",
-    "vuex": "^3.4.0"
+    "vue-router": "^3.5.1",
+    "vuetify": "^2.4.8",
+    "vuex": "^3.6.2"
   },
   "devDependencies": {
     "@vue/cli-plugin-babel": "~4.3.0",
     "@vue/cli-plugin-eslint": "~4.3.0",
-    "@vue/cli-plugin-router": "^4.3.1",
-    "@vue/cli-plugin-unit-jest": "^4.3.1",
-    "@vue/cli-plugin-vuex": "^4.4.5",
+    "@vue/cli-plugin-router": "^4.5.12",
+    "@vue/cli-plugin-unit-jest": "^4.5.12",
+    "@vue/cli-plugin-vuex": "^4.5.12",
     "@vue/cli-service": "~4.3.0",
-    "@vue/test-utils": "^1.0.3",
+    "@vue/test-utils": "^1.1.3",
     "babel-eslint": "^10.1.0",
     "eslint": "^6.7.2",
     "eslint-plugin-testcafe": "^0.2.1",
     "eslint-plugin-vue": "^6.2.2",
-    "sass": "^1.19.0",
+    "sass": "^1.32.8",
     "sass-loader": "^8.0.0",
-    "testcafe": "^1.8.5",
-    "vue-cli-plugin-vuetify": "~2.0.5",
-    "vue-template-compiler": "^2.6.11",
-    "vuetify-loader": "^1.3.0"
+    "testcafe": "^1.13.0",
+    "vue-cli-plugin-vuetify": "^2.0.9",
+    "vue-template-compiler": "^2.6.12",
+    "vuetify-loader": "^1.7.2"
```

Comment:
Did you re-run the test harness?

Reply:
Yes tests still work. (fixed one test due to an ID change I made)